realtime-face-recognition/notebooks/nuclio-face-prediction.ipynb

```diff
@@ -17,13 +17,61 @@
   },
```

Comment:
<p>Please re-run or clear outputs from this cell.</p><p> </p><p>We generally don't show pip install / function build and ofc error outputs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/demos/pull/87/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
@zilbermanor done removed all output from notebook nuclio-face-prediction.ipynb